### DISCUSSION TITLE
[Feature Request] Ctrl+Num to navigate to certain chat

### DISCUSSION THREAD

**Discussion by yoavhermon** [cstayyab/WALC#29] [Feature Requests]
[Feathub Feature Link](https://feathub.com/cstayyab/WALC/+8)

Pretty self-explanatory: Just want the ability to press ctrl+number to navigate chats directly from the keyboard. 
This worked well and was really helpful back when I used the windows client.

**Comment by cstayyab** [comment]
Thanks for suggesting this Idea. It can be implemented but it will only work from Ctrl+1 to Ctrl+9. And I guess that would  initially be enough.

**Comment by yoavhermon** [comment]
More than enough, thanks for the reply!
